README.md

```diff
@@ -32,14 +32,22 @@ load(
     "@io_bazel_rules_dotnet//dotnet:csharp.bzl",
     "csharp_repositories",
     "csharp_configure",
+    "nuget_package",
 )
 csharp_repositories()
 csharp_configure()
+
+nuget_package(
+  name="some_name",
+  pkg_name="Some.Package",
```

Comment:
I'm not sure what the prevailing style is here. I have no strong preferences myself. Does the bazel community have a style for this sort of name?